### DISCUSSION TITLE
Road closures for motor bikes (Strassensperrungen für Motorräder)   Display permanent on Cruiser Map 🙂🙂🙂

### DISCUSSION THREAD

**Discussion by DIrgi** [devemux86/cruiser#487] [Show and tell]
First of all, I don't know if this topic is already written somewhere but either I'm to stupid to form the right questions or it does not exist. I always search für catchword to see if it's already existing. So please excuse me if this topic is already somewhere.

In Germany, Austria, Switzerland - shortly the **DACH region**  (probably in some other countries as well) 
there are for several years temporary or seasonal road closures existing, only for motor bikes.
It's not really funny when you drive a tour and then the road is blocked, only for you because you are a biker additionally, your way around is many kilometers and takes some time. Luckily when you are aware about and can drive the road at the right time or have the possibility to bypass.

Unfortunately (afaik), Cruiser doesn't offer so far to have own map overlays (correct me when I'm wrong) - But cruiser can handle POIs 🙂 - So let's use the POIs to display road closures and their regulations in the map.

And to be aware what I'm talking about, there are a couple of pages handling this topic and they have often in common to display it on a google maps what we cannot use unfortunately but here's an example link how it looks like and what I'm talking about - check it out.

[Link](https://www.google.com/maps/d/viewer?mid=12cGBwKzBfUkawiT2pbgPepmGPUk&hl=en_US&ll=47.96451688773603%2C9.455719411045127&z=9)
![image](https://github.com/devemux86/cruiser/assets/57968526/c3db6353-c9e4-47d8-ade9-ed8193d609ba)

<hr>

Information about - and even as usable *.gpx file can be found via google (this is what I did). So the material I offer you here for download is not from me, I just downloaded it as well. You can find it i.e. on this page by [Christian Mintel](https://www.mintelonline.de/streckensperrungen/)
Keep this page in mind you'll find always the latest updates there. It's German but you can either translate or go to the bold **Download** and click there on the red colored **gpx-Datei** (So far, Cruiser can handle the file 1:1 so you need just to import 🙂)

<hr>

This is now the way, it looks in the Cruiser App, when you import them as POIs
(I explain to you later the setup)

What you see here is an overview of existing road closure regulations displayed in cruiser

<img width="540" alt="image" src="https://github.com/devemux86/cruiser/assets/57968526/e5dc8b61-8fb4-4048-9d7c-78bd0f624546">

<br><br><br>
When you click on a symbol you receive more detailed information. Although it's German it should be easy to understand.
First figures are local zip code. SaSo = SamstagSonntag = Saturday,Sunday road ist closed for bikes to/from Feldberg 

<img width="539" alt="image" src="https://github.com/devemux86/cruiser/assets/57968526/58703df1-b96b-4c83-aaf8-a2bacc8ce18c">

<br><br><br>
This  next map cutout is from Austria, there we have the compete year which is "Voll" mean fully or totally a complete closure for bikes louder in the papers then 95db(A) on the road from/to Bichlbach. Finally it's not 365 days closed but generally in the bikers season from spring-to autumn and most likely the rest of the year the road ist closed anyway because of winter conditions.

![image](https://github.com/devemux86/cruiser/assets/57968526/ecfca29c-c318-4eae-9712-4964145c43b6)

<hr>

So: How to proceed and bring this to Cruiser?

1. Download the attached zip file and extract, you will find a *,gpx file, containing the necessary info.
2. Open Cruiser, go to your Places, Favorites, click on the right 3 dots and select "new group" - You can give the Group any naming.
3. You see now a new group. Click on the 3 dots of this group again and select import. Select now the *.gpx file for import. Cruiser ask you to refresh the group. Select this option and then the data will be in your group, having the filename as group name. (you can always rename or change things afterwards.
4. Click now again on the tree dots and select symbol (for the POIs) - I recommend the motorbike symbol
5. To make it more dramatic and visible, click again the 3 dots and select color and here I would recommend red.
6. Last but not lest, click the 3 dots again and select: Show on Map. Activate now: Show on map and use while navigation. You can as well adjust the zoom level (btw. I had them red circled in the pictures above) I would recommend 10 as zoom level but it's up to you. Once again, you can always change your selections afterwards.

Done - now you are ready and your POIs (as shown) are displayed on the map. 🙂🙂🙂

What else... well...  I make no secret of it...my main planning tool is still Kurviger. I plan there my trips for years and via my Google Cloud I can easy transport the trips to my Cruiser App. However, if you like to have the road closures as well in Kurviger you can do the similar way.  Open Kurviger, select Cloud and Favorites. Create a new plug-in folder. Now: (a little bit different then the route import in Kurviger) click on the new favorites folder and open it. once you are done on the left side in the folder is a option symbol (with sliders). Click on it, select import and import the same gpx file I provide to you. Et voila, you have the regulations as well visible in Kurviger while planning. 

**POIs as well in Kurviger?**
One last thing you have to know is: When you download future updates from Christian Mintels page (named above), I already mentioned they'll work within Cruiser 1:1 but....not within Kurviger. I tried, but Kurviger always crashed while I tried to import.
But I have a solution, Download the file and open it with a text editor (I recommend the free Notepad++)
Now delete all entries which start with <rte> and end with </rte> (including <rte></rte) this should be 80-90% of the complete content and they are attached at the end of the document. First come the relevant POI definitions. <rte> are the coordinates for sketching the track. But POIs are not tracks and this seems to much for Kurviger and it crashes.
Once you are ready, safe the file again best with a ne name, now import into kurviger should work. btw. this is the same file I share here with you, there are only the POIs in so both can use it Cruiser and Kurviger.

Have fun, Cheers

[Streckensperrungen_Motorrad POI.zip](https://github.com/devemux86/cruiser/files/14920115/Streckensperrungen_Motorrad.POI.zip)

**Comment by devemux86** [comment]
@DIrgi many thanks!

Impressive description and details!
It really shows that you are exploring the features of Cruiser in depth. :+1: 

---

By the way, we are discussing what else we can show in route planning here:
- https://github.com/devemux86/cruiser/discussions/442

**Comment by bios4** [comment]
Congratulations @DIrgi for that excellent post! 👍

This is a very good example of how a [Cruiser manual](https://github.com/devemux86/cruiser/discussions/480) / how-to could look like...

**Comment by r11d11** [comment]
Very good description from @DIrgi. This is a great guide.
I imported the POI and it worked perfect 👍

**Comment by DIrgi** [comment]
The discussion on tread has has been moved into a collection. You'll find it here
[https://github.com/devemux86/cruiser/discussions/488](https://github.com/devemux86/cruiser/discussions/488)
